WARNING:api.v1.views:['Traceback (most recent call last):\n', '  File "/root/GitHub/new/leaseapi/leaseapi/api/v1/views.py", line 114, in create\n    print(\'all_resources=\'+repr(Resource.objects.all()))\n', '  File "/usr/local/lib/python2.7/dist-packages/django/db/models/query.py", line 138, in __repr__\n    data = list(self[:REPR_OUTPUT_SIZE + 1])\n', '  File "/usr/local/lib/python2.7/dist-packages/django/db/models/query.py", line 162, in __iter__\n    self._fetch_all()\n', '  File "/usr/local/lib/python2.7/dist-packages/django/db/models/query.py", line 965, in _fetch_all\n    self._result_cache = list(self.iterator())\n', '  File "/usr/local/lib/python2.7/dist-packages/django/db/models/query.py", line 254, in iterator\n    for row in compiler.results_iter(results):\n', '  File "/usr/local/lib/python2.7/dist-packages/django/db/models/sql/compiler.py", line 800, in results_iter\n    row = self.apply_converters(row, converters)\n', '  File "/usr/local/lib/python2.7/dist-packages/django/db/models/sql/compiler.py", line 784, in apply_converters\n    value = converter(value, expression, self.connection, self.query.context)\n', '  File "/usr/local/lib/python2.7/dist-packages/django/db/backends/mysql/operations.py", line 200, in convert_uuidfield_value\n    value = uuid.UUID(value)\n', '  File "/usr/lib/python2.7/uuid.py", line 135, in __init__\n    int = long(hex, 16)\n', "ValueError: invalid literal for long() with base 16: '<function uuid4 at 0x7f58f3a10f5'\n"]
